code/SINGE_GLG_Test.m

```diff
@@ -68,14 +68,26 @@ function SINGE_GLG_Test(Data,varargin)
 if params.prob_zero_removal~=0
    m.Xdrop = dropZeroSamples(params.prob_zero_removal, m);
 else
-    m.Xdrop = false(size(m.X));
+   m.Xdrop = false(size(m.X));
 end
 if params.replicate>0
    m.Xdrop = dropSamples(params.prob_remove_samples,m);
 end
+if ismember('targetix',who(m))
+    targetix = m.targetix;
+    if iscolumn(targetix)
+        targetix = targetix';
+    elseif ~isrow(targetix)
+        display('Error: Regulator indices must be a vector');
```

Comment:
Do we need to stop execution here instead of only displaying a message?  Or notify the user we are using all genes because their target index was improperly formatted?  Currently we won't hit the code in the `else` block that sets the target index to be all genes.

Also, I believe we want to say "Target indices" not "Regulator indices".

Reply:
Changed it to Target indices. I think it's best to error out here especially if this were to use expensive compute resources.